File: backend/src/main/java/com/gdje_izlazimo/project/dto/response/VenueImageResponse.java
package com.gdje_izlazimo.project.dto.response;

import java.time.LocalDateTime;
import java.util.UUID;

public record VenueImageResponse(
        UUID id,
        UUID venueId,
        String imageUrl,
        boolean isPrimary,
        LocalDateTime createdAt,
        LocalDateTime updatedAt
) {
}

File: frontend/src/app/core/models/venue-images/venue-image-response.ts
// src/app/core/models/venue-images/venue-image-response.dto.ts
export interface VenueImageResponseDto {
  id: string;
  venueId: string;
  imageUrl: string;
  isPrimary: boolean;
  createdAt: string;
  updatedAt: string;
}